include/z64actor.h

```diff
@@ -199,31 +200,50 @@ typedef struct {
     /* 0x162 */ s16 unk_162;
 } DynaPolyActor; // size = 0x164
 
+typedef struct {
+    /* 0x00 */ s32 active;
+    /* 0x04 */ Vec3f tip;
+    /* 0x10 */ Vec3f base;
+} Struct_80090480_arg2;
+
 typedef struct {
     /* 0x0000 */ Actor      actor;
     /* 0x014C */ s8         currentTunic;
     /* 0x014D */ s8         currentSword;
     /* 0x014E */ s8         currentShield;
     /* 0x014F */ s8         currentBoots;
-    /* 0x0150 */ s8         unk_150;
-    /* 0x0151 */ s8         unk_151;
+    /* 0x0150 */ s8         heldItemCButtonIdx;
+    /* 0x0151 */ s8         heldItemActionParam;
     /* 0x0152 */ s8         unk_152;
     /* 0x0153 */ s8         unk_153;
     /* 0x0154 */ s8         unk_154;
-    /* 0x0155 */ char       unk_155[0x008];
+    /* 0x0155 */ char       unk_155[0x003];
+    /* 0x0158 */ s8         unk_158;
+    /* 0x0159 */ char       unk_159[0x002];
+    /* 0x015B */ u8         unk_15B;
+    /* 0x015C */ u8         unk_15C;
     /* 0x015D */ u8         unk_15D;
     /* 0x015E */ u8         unk_15E;
     /* 0x015F */ u8         currentMask;
-    /* 0x0160 */ char       unk_160[0x050];
-    /* 0x01B0 */ u32        unk_1B0;
+    /* 0x0160 */ UNK_PTR    unk_160;
+    /* 0x0164 */ UNK_PTR    unk_164;
+    /* 0x0168 */ UNK_PTR    unk_168;
+    /* 0x016C */ UNK_PTR    unk_16C;
+    /* 0x0170 */ u8         unk_170;
+    /* 0x0171 */ char       unk_171[0x023];
+    /* 0x0194 */ OSMesgQueue unk_194;
+    /* 0x01AC */ char       unk_1AC[0x004];
+    /* 0x01B0 */ u32        getItemModelPtr;
```

Comment:
should this be a pointer type? If they type is u32* id also think that the word Ptr in the name is unnecessary but that might just be me